doc/README.Solaris

```diff
@@ -19,6 +19,15 @@ Building from source
 Please make sure all required dependencies are installed. On Solaris 11,
 this would be:
     pkg install autoconf libtool pkg-config automake gcc asciidoc docbook
+on Omnios:
+    pkg install developer/build/autoconf \
+                developer/build/libtool \
+                developer/pkg-config \
+                developer/build/automake \
+                developer/gcc4.8 \
```

Comment:
Are you sure? I think it should be developer/gcc48 (without the dot), at least that's how I made it work for me.